templates/services/merge/execution/besu.tmpl

```diff
@@ -0,0 +1,41 @@
+{{/* besu.tmpl */}}
+{{ define "execution" }}
+  execution:
+    image: ${EC_IMAGE_VERSION}
+    container_name: execution_client
+    networks:
+    - sedge
+    user: root
+    ports:
+      - "{{.ElDiscoveryPort}}:{{.ElDiscoveryPort}}/tcp"
+      - "{{.ElDiscoveryPort}}:{{.ElDiscoveryPort}}/udp"
+      - "{{.ElMetricsPort}}:{{.ElMetricsPort}}/tcp"{{if .MapAllPorts}}
+      - "{{.ElApiPort}}:{{.ElApiPort}}"
+      - "{{.ElAuthPort}}:{{.ElAuthPort}}"{{end}}
+    expose:
+      - {{.ElApiPort}}
+      - {{.ElAuthPort}}
+    volumes:
+      - ${EC_DATA_DIR}:/var/lib/besu/data
+      - ${CC_JWT_SECRET_PATH}:/var/lib/besu/jwtsecret
+    command:
+    - --sync-mode=X_SNAP
+    - --data-storage-format=BONSAI
+    - --network={{if .SplittedNetwork}}${EL_NETWORK}{{else}}${NETWORK}{{end}}
+    - --data-path=/var/lib/besu/data
+    - --metrics-enabled=true
+    - --metrics-host=0.0.0.0
+    - --metrics-port={{.ElMetricsPort}}
+    - --engine-rpc-enabled=true
+    - --engine-jwt-secret=/var/lib/besu/jwtsecret
+    - --engine-rpc-port={{.ElAuthPort}}
+    - --engine-host-allowlist=*
+    - --rpc-http-enabled=true
+    - --rpc-http-host=0.0.0.0
+    - --rpc-http-port={{.ElApiPort}}
+    - --rpc-http-api=ETH,NET,CLIQUE,DEBUG,MINER,NET,PERM,ADMIN,EEA,TXPOOL,PRIV,WEB3
+    - --rpc-http-cors-origins=*
+    - --host-whitelist=*
+    restart: unless-stopped
+    mem_swappiness: 0
```

Comment:
I believe putting `mem_swappiness` here and not within a `resources` property is deprecated in Compose v3.0. Is there a reason why we should configure this?

Reply:
No reason, deleting it now